### ISSUE TITLE
Build Documentation Incorrect

### ISSUE THREAD

**Issue by tobine** [google/agi#254]
The AGI Windows build instructions under [Install Bazel](https://github.com/google/agi/blob/master/BUILDING.md#install-bazel) state:

> Note: Installing bazel will also install MSYS into C:\tools\msys64 and Python into C:\tools\python27.

However, I successfully followed the instructions to run `choco install bazel` and bazel is installed without having msys64 or python27 installed in the c:\tools dir.

**Comment by hevrard** [comment]
Note on this: now that we're using Bazel 2.0.0, we don't need Python2.7 as a dependency any more (it was required for the previous version of Bazel we used).

**Comment by pau-baiget** [comment]
I've got a question regarding Python dependency. Installed Python 3.8 in my windows desktop and I was getting the following errors while building:

```bazel build pkg
Starting local Bazel server and connecting to it...
ERROR: C:/development/agi-2/agi/core/vulkan/vk_virtual_swapchain/cc/BUILD.bazel:75:1: in strip rule //core/vulkan/vk_virtual_swapchain/cc:libVkLayer_VirtualSwapchain_stripped.dll:
Traceback (most recent call last):
        File "C:/development/agi-2/agi/core/vulkan/vk_virtual_swapchain/cc/BUILD.bazel", line 75
                strip(name = 'libVkLayer_VirtualSwapchain_stripped.dll')
        File "C:/development/agi-2/agi/tools/build/rules/cc.bzl", line 62, in _strip_impl
                ctx.actions.run(<4 more arguments>)
        File "C:/development/agi-2/agi/tools/build/rules/cc.bzl", line 63, in ctx.actions.run
                cc_toolchain.strip_executable()
'string' object is not callable
ERROR: Analysis of target '//:pkg' failed; build aborted: Analysis of target '//core/vulkan/vk_virtual_swapchain/cc:libVkLayer_VirtualSwapchain_stripped.dll' failed; build aborted
INFO: Elapsed time: 5.722s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (84 packages loaded, 1829 targets configured)
    Fetching @androidndk; Restarting.
```

I fixed that with this patch that I apply locally:

```
diff --git a/gapidapk/android/apk/rules.bzl b/gapidapk/android/apk/rules.bzl
index 81e28920..5de1c921 100644
--- a/gapidapk/android/apk/rules.bzl
+++ b/gapidapk/android/apk/rules.bzl
@@ -21,7 +21,7 @@ def _strip_impl(ctx):
     if cc_toolchain.cpu == ctx.attr.abi:
         out = ctx.actions.declare_file("lib/{}/{}".format(ctx.attr.abi, ctx.file.lib.basename))
         ctx.actions.run(
-            executable = cc_toolchain.strip_executable(),
+            executable = cc_toolchain.strip_executable,
             arguments = ["--strip-unneeded", "-o", out.path, ctx.file.lib.path],
             inputs = [ctx.file.lib] + ctx.files._ndk,
             outputs = [out],
diff --git a/tools/build/rules/cc.bzl b/tools/build/rules/cc.bzl
index a4d27d2f..19ca1faf 100644
--- a/tools/build/rules/cc.bzl
+++ b/tools/build/rules/cc.bzl
@@ -59,8 +59,9 @@ def _strip_impl(ctx):
     else:
         fail("Unhandled CPU type in strip rule: " + cc_toolchain.cpu)

-    ctx.actions.run(
-        executable = cc_toolchain.strip_executable(),
+    ctx.actions.run(
+        executable = cc_toolchain.strip_executable,
         arguments = flags + ["-o", out.path, src.path],
         inputs = [src],
         outputs = [out],
--
```

Since the installed Python version does actually affect the build, are you sure we don't need Python anymore?

**Comment by hevrard** [comment]
Are you using Bazel 2.0.0 ?

**Comment by tobine** [comment]
Note that when installing bazel using the build [instructions](https://github.com/google/agi/blob/master/BUILDING.md#install-bazel) I got 3.1.0 by default. If 2.2.0 is recommended version, should update instructions to state `choco install bazel --version 2.2.0`

**Comment by hevrard** [comment]
Hmm, I think we should drop the Choco thing entirely, if possible. It seems like it is creating more issues than helping.

The real windows build workflow that is tested is the one we use for the CI:
https://github.com/google/agi/blob/master/kokoro/windows/build.bat

I think it would be good to align the "BUILDING.md" documentation to the flow used for the CI.

**Comment by hevrard** [comment]
So it looks like:
- the doc inherited from the time we used Bazel 1.0.0 says that installing Bazel via chocolatey would also install Python and Msys, because Bazel 1.0.0 was depending on them:
https://chocolatey.org/packages/bazel/1.0.0#dependencies

- since we updated to Bazel 2.0.0, I think the Bazel package depends on neither Python nor MSYS2:
https://chocolatey.org/packages/bazel/2.0.0#dependencies

At this point, I think it might be best to tell our users to use chocolatey to install:
- bazel **2.0.0**
- MSYS2
- Python2.7

(I don't have a windows machine to try and find the exact commands)

**Comment by pau-baiget** [comment]
Fixed by  #297
